libs/net/models/Settings/ChartTemplateSettingsModel.cs

```diff
@@ -0,0 +1,30 @@
+using System.Text.Json;
+using TNO.Models.Extensions;
+
+namespace TNO.API.Models.Settings;
+
+public class ChartTemplateSettingsModel
```

Comment:
All of the models in this section are for JSON fields in the database.  It was an absolute pain to get all this working due to case-sensitivity.